### DISCUSSION TITLE
Invidious Team being contact by Youtube

### DISCUSSION THREAD

**Discussion by tukangcode** [TeamNewPipe/NewPipe#10152] [Questions] [answered]
Disc : https://github.com/iv-org/invidious/issues/3872
Basicly youtube claiming invidious violating their API term which in fact invidious not use any youtube api. i bit worried they will start crackdown on you so what you view on this ?

**Comment by Stypox** [comment, marked as answer]
Yes we saw it. We are sharing our thoughts on the IRC room.
